### ISSUE TITLE
Conflicts when using typescript

### ISSUE THREAD

**Issue by yyf401238353** [proyecto26/ion-phaser#8]
It seems like your code in @ion-phaser/react/dist/types/components/IonPhaser.d.ts overwrites the JSX in @type/react.

**Comment by yyf401238353** [comment]
declare module "react" {
    namespace JSX {
        interface IntrinsicElements {
            'ion-phaser': any;
        }
    }
}

**Comment by jdnichollsc** [comment]
Yes here https://github.com/proyecto26/ion-phaser/blob/master/react/src/components/IonPhaser.tsx#L4

**Comment by jdnichollsc** [comment]
oh ok, we need to replace that by using **global** instead:
```
declare global {
  namespace JSX {
    interface IntrinsicElements {
      'ion-phaser': any;
    }
  }
}
```

**Comment by jdnichollsc** [comment]
Fixed, check the last version 1.2.1! 👍
